packages/sidesail/Justfile

```diff
@@ -0,0 +1,2 @@
+lint:
+   find . -name "*.dart" -not -path "./lib/gen/*" | xargs dart format -l 120
```

Comment:
foooormatting

Reply:
would it be better to have a dart formatting task in a root `Justfile`?

Reply:
not yet. Maybe later if we want to get even fancier. most of the time you work with one specific package